### DISCUSSION TITLE
Использование HAproxy вместо Keepalived.

### DISCUSSION THREAD

**Discussion by rickyticky2** [autobase-tech/autobase#157] [General]
Добрый день, спасибо за ценный репозиторий. 
  В облаке , что я испозую, нет возможности использовать протокол vrrp . Возможно ли использование дополнительного сервера с haproxy, для проверки состояния других серверов с Haproxy ? 
И вообще, как считаете ,  на сколько целесообразно такое решение ? (мне будет удобно упаковать доп Haproxy  в контенер и положить его в класстер кубера )
ПС. по типо B таким способом уже простестируовал , все ОК. Примерно с таким конфигом : 

```
listen postgres
    bind *:5000
    option httpchk
    http-check expect status 200
    default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
    server postgresql1 10.129.0.1:6432 maxconn 100 check port 8008
    server postgresql2 10.129.0.2:6432 maxconn 100 check port 8008
    server postgresql3 10.129.0.3:6432 maxconn 100 check port 8008
```

**Comment by vitabaks** [comment]
Добрый день. 
Да, можите. Haproxy не обязательно должен быть установлен на серверах баз данных. Но убедись что между сервером haproxy и postgresql нет высокого latency.

**Reply by rickyticky2** [reply]
Спасибо за быстрый ответ ) а не подскажите , какой обьем дисков нужен для etcd ? планирую запросить диски отдельно , только для DCS .

**Reply by vitabaks** [reply]
10GB на SSD будет достаточно. DCS обычно занимает мегабайты, там хранится очень мало информации.

**Reply by vitabaks** [reply]
данные с пром кластера (из 3 нод Patroni и 5 нод etcd) работающего с 2018 года
```
sudo du -sh /var/lib/etcd/
248M    /var/lib/etcd/
```
